src/pages/instances/actions/snapshots/CreateSnapshotForm.tsx

```diff
@@ -47,6 +69,14 @@ const CreateSnapshotForm: FC<Props> = ({ instance, close }) => {
   const SnapshotSchema = Yup.object().shape({
     name: Yup.string()
       .required("This field is required")
+      .test("deduplicate", "Snap name is already in use", (value) =>
```

Comment:
"Snapshot name" or "Name"? why "Snap name" ...

Reply:
This is the label we have in the HFM currently, should it be changed @al-bakalova?

Reply:
Ok, done.